LDAR_Sim/model_code/weather_lookup.py

```diff
@@ -34,9 +34,9 @@ def __init__(self, state, parameters):
 
         # Read in weather data as NetCDF file(s)
 
-        self.wind_temp_data = Dataset(self.parameters['working_directory'] + self.parameters['an_data'])  # Load wind and temp data
+        self.wind_temp_data = Dataset(self.parameters['working_directory'] + self.parameters['era5_data'])  # Load wind and temp data
```

Comment:
@AkaMrmuffin This looks like it will work but do we need different file names now or should we combine everything into a single file (like, instead of having wind_temp_data and precip_data, should we just heave weather_data)?